transport-classes-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -355,6 +364,36 @@ static void checkKernelVersion(String kernelVersion) {
         }
     }
 
+    static boolean checkKernelVersion(String kernelVersion, int major, int minor) {
```

Comment:
You can remove `checkKernelVersion0` and just use this one from `checkKernelVersion(String)`